tensorflow/core/util/cuda_kernel_helper.h

```diff
@@ -26,6 +26,16 @@ limitations under the License.
 #include "tensorflow/core/platform/stream_executor.h"
 #include "tensorflow/core/platform/types.h"
 
+#if defined(__CUDACC_VER_MAJOR__) && __CUDACC_VER_MAJOR__ < 9
```

Comment:
TensorFlow defines a CUDA_VERSION macro; please use that so this code will be portable to other compilers.

Reply:
TF_CUDA_VERSION is a string containing a decimal value (e.g. "8.0") rather than an INT. This makes it unwieldy for inequality comparisons.

Reply:
What about the CUDA_VERSION macro?  https://github.com/tensorflow/tensorflow/search?utf8=%E2%9C%93&q=cuda_version&type=

Reply:
Perhaps I'm being dense, but I don't see where that macro is defined, and it doesn't seem to be available in CUDA compilation units.

Reply:
CUDA_VERSION is defined by nvidia's headers, in cuda.h.

(FYI I'm going to be OOO for a week; hopefully you can get assistance from the TF team in the meantime.)